crates/forge/tests/fixtures/zk/TwoUserMultisig.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: MIT
```

Comment:
It feels we could make this contract much simpler as the logic here is not that important to the things we are testing. Maybe change this to the simplest version possible of a smart account (it could even be the default one)